spec/feature_flagger/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Configuration do
+    context 'accessors methods' do
+      subject(:configuration) { described_class.new }
+
+      it { expect(configuration).to respond_to(:storage) }
+      it { expect(configuration).to respond_to(:storage=) }
+      it { expect(configuration).to respond_to(:yaml_filepath) }
+      it { expect(configuration).to respond_to(:info) }
+    end
```

Comment:
You could create a double in your test file 
```ruby
Rails = double(root: '/')
```

But I think this Rails defined logic isn't on the right place. The configuration doesn't need to know about rails existence. As the configuration depends of the yml path, you should pass it as an argument on initialize and do this logic on the main module

```riby
def config
  @configuration ||= Configuration.new(yml_path)
end
```